frontend/js/components/FeatureGrid.jsx

```diff
@@ -488,11 +506,69 @@ class SiraGrid extends React.Component {
             url + "&FEATUREID=" + params.data.id + (this.props.params.authkey ? "&authkey=" + this.props.params.authkey : "")
         );
 
+        params?.data?.geometry?.coordinates && this.props.setGeometry(params?.data?.geometry);
+
         if (!this.props.detailOpen) {
             this.props.onShowDetail();
         }
     };
 
+    multiLayerSelect = (params) => {
+        this.props.setTreeFeatureType(undefined);
+        this.props.closeTree();
+        const filterObj = {
+            groupFields: [],
+            filterFields: [],
+            spatialField: {}
+        };
+        const zoomEnabled = params.data?.geometry?.coordinates;
+        this.props.configureMLS(filterObj, zoomEnabled);
```

Comment:
const value = ___extract value for configured attribute path from the main feature___
this.props.configureMLS(value, zoomEnabled);